hannakim91/travel-tracker
# Dom/traveler dashboard 2

# Travel Tracker Pull Request

### What’s this PR do?  
Build out UI for Traveler Dashboard:
- Traveler can see their past/upcoming trips
- Traveler can see their spendings in 2019 and 2020
- Traveler can see estimates for and book a new trip, which will show up as a pending trip in their trips section above
- Have code ready to `DELETE` trip instances after I POSTed something that did not have all trip info and was causing bugs -- may implement on `Traveler` side since I don't think I'll have time to implement `Agent` features
### Where should the reviewer start?  
- index.js
- fetch.js
- domUpdates.js
- Trip.js/Trip-test.js
- index.html
### How should this be manually tested?  
Run `npm start` and open localhost:8080 in a browser to see functions
**Note: No limitations/stopgaps on "Book Trip" button -- only POST with all info relevant to project for now**
### Any background context you want to provide?  
Finishing up features for Iteration 1+2 on spec
### What are the relevant tickets?  
Resolves #9;
Resolves #10;
Resolves #11;
Resolves #13;
Resolves #14;
Resolves #15;
Resolves #16; 
Resolves #48;
Resolves #54;
Resolves #58; 

### Screenshots (if appropriate)  

### Questions: 
 #53; **note** currently giving random 4-digit ID's - will ask instructor
#63; left alone for now b/c it makes more sense to leave it in index.js and reuse inputs from form
